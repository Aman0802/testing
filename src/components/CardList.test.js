import { shallow } from "enzyme";
import React from "react";
import CardList from "./CardList";

it("expect CardList component to rendr", () => {
  const mockRobots = [
    {
      id: 1,
      name: "Aman Khubani",
      email: "aman.khubani@gmail.com",
      username: "aman0802",
    },
  ];
  expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
});
