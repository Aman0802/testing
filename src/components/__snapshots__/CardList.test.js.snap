// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect CardList component to rendr 1`] = `ShallowWrapper {}`;
